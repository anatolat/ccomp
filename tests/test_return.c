
int test_i() {
	printf("test_i\n");
	return 42;
	printf("test_i\n");
}

void test_void() {
	printf("test_void\n");
	return;
	printf("test_void\n");
}

int main() {
	printf("%d\n", test_i());
	test_void();
}