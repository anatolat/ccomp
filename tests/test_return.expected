test_i
42
test_void
